library/src/G4LatticeReader.cc

```diff
@@ -392,12 +395,31 @@ G4bool G4LatticeReader::ProcessEulerAngles(const G4String& name) {
     G4cerr << "G4LatticeReader: Unknown rotation matrix " << name << G4endl;
     return false;
   }
-
+    
   G4double degOrRad = ProcessUnits("Angle");
   pLattice->AddValley(phi*degOrRad, theta*degOrRad, psi*degOrRad);
+  // Anti-valleys
+  pLattice->AddValley(phi*degOrRad+pi, -1*(theta*degOrRad+pi), -psi*degOrRad);
```

Comment:
Isn't the second one equivalent to just -theta*degOrRad+pi (since +pi and -pi are equivalent)?

Reply:
you are right! Changed it to : -theta*degOrRad+pi